ui/src/main/js/page/channel/azure/AzureBoardTable.js

```diff
@@ -41,7 +41,7 @@ const emptyTableConfig = {
     message: 'There are no records to display for this table.  Please create an Azure Board connection to use this table.'
 };
 
-const AzureBoardTale = ({ readonly, allowDelete }) => {
+const AzureBoardTable = ({ readonly, allowDelete, location }) => {
```

Comment:
Can location be directly retrieved in this component via useLocation? Meaning moving useLocation from AzureBoardsLayout to AzureBoardsTable